app/code/Magento/Catalog/ViewModel/Product/Breadcrumbs.php

```diff
@@ -101,15 +107,14 @@ public function getProductName(): string
      */
     public function getJsonConfigurationHtmlEscaped() : string
     {
-        return json_encode(
+        return $this->jsonSerializer->serialize(
             [
                 'breadcrumbs' => [
                     'categoryUrlSuffix' => $this->escaper->escapeHtml($this->getCategoryUrlSuffix()),
                     'useCategoryPathInUrl' => (int)$this->isCategoryUsedInProductUrl(),
                     'product' => $this->escaper->escapeHtml($this->getProductName())
                 ]
-            ],
-            JSON_HEX_TAG
+                ]
```

Comment:
Looks like formatting is little bit broken. Please fix it
```suggestion
            ]
```

Reply:
@ihor-sviziev updated